src/registration_and_login/LoginForm.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import validator from 'validator';
+import { isEmail, isLength } from 'validator';
+
+const LoginForm = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isValidEmail, setIsValidEmail] = useState(true);
+  const [error, setError] = useState('');
+  const [status, setStatus] = useState('Not loged');
+
+  const handleEmailChange = (event) => {
+    const newEmail = event.target.value;
+    setEmail(newEmail);
+    setIsValidEmail(validator.isEmail(newEmail));
+  };
+
+  const handleSubmit = async (event) => {
```

Comment:
Ináč ten `onSubmit` event má v sebe dáta o všetkých inputoch ktoré v sebe daný `<form>` má:
```js
<form onSubmit={e => {
  const aaaValue = e.target.aaa.value
  const bbbValue = e.target.bbbb.value
  
  // Alebo sa dajú hodnoty všetkých inputov (ktoré majú name atribút) vycucnúť dynamicky,
  // bez toho aby si musel referovať konkrétne mená
  
  // FormData je špeciálny typ objektu, ktorý v sebe má informácie o formuláry,
  // ktorý dostane do constructora.
  // Nedá sa s ním však pracovať ako s normálnym JS objektom,
  // ale dá sa do normálneho objektu skonvertovať...
  const formData = new FormData(e.target)
  // ...toto nám dá iterátor pomocou ktorého môžme listovať cez jednotlivé inputy a ich hodnoty,
  // napríklad cez for loop...
  const formDataIterator = formData.entries()
  // ...alebo cez array spread, ako tu
  const entries = [...formDataIterator]
  // výsledok bude [['aaa', 'hodnota inputu a'], ['bbb', 'hodnota inputu b']],
  // ktorý potom môžme premeniť na normálny objekt takto
  const formDataPlainObject = Object.fromEntries(entries)
 
  // Ale dá sa to celé napísať v jednom riadku
  const values = Object.fromEntries(new FormData(e.target).entries())
  // výsledok bude {aaa: 'hodnota inputu a', bbb: 'hodnota inputu b'}
}}>
  <input name="aaa"/>
  <input name="bbb"/>
</form>
```